### ISSUE TITLE
App fails to warn

### ISSUE THREAD

**Issue by TheLastProject** [minvws/nl-covid19-notification-app-android#67]
### Describe the bug, issue or concern

Apparently I've been close to someone, but the app literally failed to warn me. The German "Corona Contact Tracing Germany" app warned me correctly though.

### To Reproduce
1. Be near someone infected for a prolonged time in the Netherlands
2. Wait for the German app to alert you
3. Notice the Dutch app act like everything is fine

### Expected behavior
The Dutch app should warn me when there is an increased risk.

### Screenshots
![image](https://user-images.githubusercontent.com/1885159/157036040-70f591c8-cf80-40d5-b41d-50d7eb7a732c.png)

**Smartphone (please complete the following information):**
 - Device: Fairphone 3
 - OS: LineageOS for microG 11
 - Version: 2.5.4 build 142096-cc48c421

### Additional context
I have never gotten any alert from the Dutch app in multiple years of usage. I contributed this to my careful behaviour. However, given I only installed the German app 7 days ago and already got a warning I now worry the Dutch app has been silently failing consistently...

CCTG noted that it was last updated today at 13:25 when it showed the risk, so I waited until the  Dutch app showed "Last checked on March 7, 14:11" before reporting this issue (screenshot taken earlier). Given I was in the Netherlands on March 4th (the date CCTG warned me for), I would expect this to be waiting long enough.

### Governance
- [x] I've read the contributing document https://github.com/minvws/.github/blob/master/CONTRIBUTING.md
- [x] I've read and understand the Code of Conduct https://github.com/minvws/.github/blob/master/CODE_OF_CONDUCT.md
- [x] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/LICENSE.txt and the contributor license agreement https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/CLA.md

**Comment by TheLastProject** [comment]
> The Exposure Notification framework allows normally to have only one covid tracing app active. When trying to activate exposure notification framework for a different app, the framework should ask you for permission to switch app.

Interesting, this must be a difference then between microG and the proprietary Google implementation, as microG seems to allow multiple at once and lacks the "Exposure notifications sent from" setting.
![image](https://user-images.githubusercontent.com/1885159/157072529-8c746100-34d5-466b-9754-8af31168072c.png)

> When CoronaMelder is active (while CCTG is deactivated) could you check if your phone runs exposure checks normally? It should do a couple of checks a day. Exposure check details can be found under Android settings > Google > Covid-19 Exposure Notifications > 3 dots in top right corner > Exposure checks. Each exposure check should contain a number of keys and could show a number of matches when you've been close to someone with Covid-19.

This also seems to differ in the microG implementation, but I can verify that microG has been collecting a lot of IDs:
![image](https://user-images.githubusercontent.com/1885159/157073707-4f999829-4145-4f77-88bd-b51747ee7b0f.png)

Also, microG reports to have sent the exact same data to both CoronaMelder and CCTG:
![image](https://user-images.githubusercontent.com/1885159/157073822-d5a2c9d5-9ab5-4023-8eed-7577199a424c.png)
![image](https://user-images.githubusercontent.com/1885159/157073835-847ee99c-9f0c-4c54-ba67-b5d93b561eec.png)

After uninstalling CCTG and rebooting my phone (just to make sure), CoronaMelder still hasn't alerted me.

> Based on the exposure, CoronaMelder should trigger a notification if the exposure is above a certain risk threshold.

It is notable that microG only logged distant exposures. Could it be that the Dutch and German app simply disagree on what risk threshold is worth warning for (specifically, CCTG considers 36 minutes of distant exposure to be problematic while CoronaMelder does not)? That would surprise me, as I would have expected the threshold to be scientifically decided in an international fashion, but does seem like the simplest explanation.

**Comment by lpappalettera** [comment]
@TheLastProject thanks for the detailed info. I will try to figure out if the risk params could be the reason for not receiving a notification (while CCTG does trigger one). Each has country has their own risk calculation to match the covid measures. In CoronaMelder we've tried to match the 1,5 meter rule with cumulative duration of at least 15 minutes. If interested you can find more info about the risk calculation here: https://developers.google.com/android/exposure-notifications/meaningful-exposures

**Comment by lpappalettera** [comment]
The difference in risk parameters could have lead to the different behaviour. On CoronaMelder basically ignore exposure with an attenuation above 73. This threshold is a little bit higher in de CCTG (see https://github.com/corona-warn-app/cwa-documentation/blob/main/images/risk_calculation/risk_calculation_enf_v2_overview.pdf).

**Comment by TheLastProject** [comment]
Well, in that case I suppose it is not a bug, just a difference in definition of "acceptable risk" between the apps.

I'll close this issue then given it's not a real issue. I'll reopen it if I ever see the Dutch app fail for something more severe than "distant exposure over 15 minutes". Thanks for your time and research, it's very appreciated, and I am glad to understand this better now. Given I tested negative after this encounter, maybe the Dutch app's risk calculation is more realistic anyway (or I just got lucky 😄)
